support/test/integration/docker-compose.yml

```diff
@@ -23,7 +23,7 @@ services:
       - ETHEREUM_HOST=parity-dev-node
       - ETHEREUM_PORT=8545
       - ETHEREUM_GAS_PRICE_IN_NANOETH=1
-      - ETHEREUM_PRIVATE_KEY=0xe2b0241b73cd78d450f85eca67188fb884e7ebe961f8981814474c500d581952
+      - ETHEREUM_PRIVATE_KEY=0xfae42052f82bed612a724fec3632f325f377120592c75bb78adfcceae6470c5a
```

Comment:
These private keys now align with the accounts that have all of the ETH in our dev nodes.  This means you can run integration tests against shared dev chains easily (currently hosted on Azure via Docker Cloud and setup to auto-build).